deploy/test/global-manifests.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: envdeployments.devconsole.openshift.io
+spec:
+  group: devconsole.openshift.io
+  names:
+    kind: DevConsole
+    listKind: DevConsoleList
+    plural: envdeployments
+    singular: envdeployment
+  scope: Namespaced
+  version: v1alpha1
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: devconsole
+rules:
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - clusterrolebindings
```

Comment:
I think we don't need our test `sa` needs to have permission for `clusterrolebindings`

Reply:
Fixed befe3ef.